src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import React from 'react';
+import cm from 'classnames';
+import { Tab } from '../../Types/Tab';
+
+type Props = {
+  tabs: Tab[]
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = (
+  { tabs, selectedTabId, onTabSelected: onSelected },
+) => {
+  const handleSelectedTab = (tab: Tab) => {
+    if (tab.id !== selectedTabId) {
+      onSelected(tab);
+    }
+  };
```

Comment:
Better way for handler
```suggestion
  const handleSelectedTab = (tab: Tab) => () => {
    if (tab.id !== selectedTabId) {
      onSelected(tab);
    }
  };
```